wireframe/style/style.css

```diff
@@ -0,0 +1,145 @@
+.main-description {
+	letter-spacing: 1px;
+}
+
+
+.border , .border-bottom , .border-top{
+	border-color: #999999;
+}
+
+hr {
+	height: .5px;
+	color: #999999;
+	background-color: #999999;
+	border: none;
+}
+
+/******** SEARCH RESULT PAGE ********/
+
+.less-line-height {
+	line-height:14px;
+}
+
+.fix-size {
+  height: 95px;
+  width: 95px
+}
+
+.picture-box {
+	height: 29px;
+}
+
+/*.circle-checkbox {
```

Comment:
please don't leave garbage code, if it works leave it, if it doesn't, remove it, that's why we have versioning :) so you can go back and see how it was